src/components/modals/data-table-modal.js

```diff
@@ -90,8 +90,19 @@ function DataTableModalFactory(DataGrid) {
     const activeDataset = datasets[dataId];
     const rows = activeDataset.data;
 
-    const columns = activeDataset.fields
-      .filter(({name}) => name !== '_geojson');
+    const {columns, dataColumnMap} = activeDataset.fields.reduce((acc, field, index) => {
+      const valid = field.name !== '_geojson';
```

Comment:
We can add a function to parse `_geojson` field instead of filter them out. 
I think it's useful to display it, especially when people uploaded csv contains `WKT` and `Feature`. Put it in `data-utils` maybe, so it can be used in other places

```js
const parseFieldValue = (value, type) => {

  if (!notNullorUndefined(value)) {
    return '';
  }

  switch (type) {
    case ALL_FIELD_TYPES.boolean:
      return String(value);
    case ALL_FIELD_TYPES.geojson:
      return typeof value === 'string' ? value :
      isPlainObject(value) ? JSON.stringify(value) :
      Array.isArray(value) ? `[${String(value)}]` : '';
    default:
      return value;
  }
}
```